### PR TITLE
Item Show Page

### PR DESCRIPTION
**Pull request by hsmitha26**
Completes User Story 16 (#67 )

Feature and model tests at 100%

- Adds functionality for any user to view details about an item on the item's show page.
- Adds model test for #average_fulfillment_time in days.

### CODE REVIEW

File: app/views/items/show.html.erb [hunk 1]
```diff
@@ -4,5 +4,6 @@
 <p>Merchant Name: <%= @merchant.name %></p>
 <p><%= @item.inventory %> in stock</p>
 <p>$<%= @item.price %></p>
+<p>Average Fulfillment Time: <%= @item.average_fulfillment_time %> days</p>
 
 <%= button_to "Add to Cart", carts_path(item_id: @item.id) %>
```
Comment (smainar) [inline app/views/items/show.html.erb]: On line 9, it needs a conditional statement for only a visitor or regular user to see a link to add this item to their cart. The admin and merchant should not see the "Add to Cart" link.

File: spec/features/items/show_spec.rb [hunk 2]
```diff
@@ -5,11 +5,15 @@
...
 
-    it "I can see all the information for the item" do
+      @order_item_1 = create(:fulfilled_order_item, item: @item_1, created_at: 3.days.ago, updated_at: 1.days.ago)
+      @order_item_2 = create(:fulfilled_order_item, item: @item_1, created_at: 2.days.ago, updated_at: 1.days.ago)
+      @order_item_3 = create(:fulfilled_order_item, item: @item_1, created_at: 4.days.ago, updated_at: 1.days.ago)
+
       visit item_path(@item_1)
+    end
 
+    it "I can see all the information for the item" do
```
Comment (smainar) [inline spec/features/items/show_spec.rb]: I would add a test for a visitor and a regular user to check that they will not see a link for "Add to Cart".

File: app/views/items/show.html.erb (hunk 1 above)
Comment (hsmitha26) [reply app/views/items/show.html.erb]: Adds condition. Thank you for catching this, Stella.

File: spec/features/items/show_spec.rb (hunk 2 above)
Comment (hsmitha26) [reply spec/features/items/show_spec.rb]: Adds test to see who can see the Add to Cart button.
Thank you, Stella!

**Pull request merged**